guides/common/modules/proc_enabling-tuned-profiles.adoc

```diff
@@ -0,0 +1,42 @@
+[id="Enabling_Tuned_Profiles_{context}"]
+= Enabling Tuned Profiles
+
+{RHEL} 7 enables the tuned daemon by default during installation.
+On bare-metal, {Team} recommends to run the `throughput-performance` tuned profile on {ProjectServer} and {SmartProxies}.
+On virtual maschines, {Team} recommdsn to run the `virtual-guest` profile.
+
+.Procedure
+. Check if `tuned` is running:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# systemctl status tuned
+----
+. If `tuned` is not running, enable it:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# systemctl enable tuned
+----
+. Optional: View a list of available `tuned` profiles:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# tuned-adm list
+----
+. Enable a `tuned` profile depending on your scenario:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# tuned-adm profile "_My_Tuned_Profile_"
+----
+
+// what does this mean? Where should it go?
+Transparent Huge Pages is a memory management technique used by the Linux kernel which reduces the overhead of using Translation Lookaside Buffer (TLB) by using larger sized memory pages.
+Due to databases having Sparse Memory Access patterns instead of Contiguous Memory access patterns, database workloads often perform poorly when Transparent Huge Pages is enabled.
+To improve performance of PostgreSQL, {Team} recommends Transparent Huge Pages be disabled.
```

Comment:
This isn't just us, it's really a best practice. https://www.percona.com/blog/2019/03/06/settling-the-myth-of-transparent-hugepages-for-databases/ actually has some nice write up.

Reply:
Reworded.